picasso/src/test/java/com/squareup/picasso/BitmapHunterTest.java

```diff
@@ -1059,7 +1060,7 @@
       BitmapHunter.applyCustomTransformations(transformations, original);
       fail("Expected exception to be thrown.");
     } catch (RuntimeException e) {
-      assertThat(e).hasMessage("Transformation "
+      assertThat(e).hasMessageThat().isEqualTo("Transformation "
```

Comment:
The other case which would be suitable for changing is this one. Instead of using `isEqualTo`, we could use `contains`.

While I was going through I noticed that `hasMessageContaing` was being used in one case, but not on the rest (which use `hasMessage`). That's why I opted to use `isEqualTo` in order to preserve the original meaning.